src/DynamoCore/Models/DynamoModelEvents.cs

```diff
@@ -73,6 +73,15 @@ internal static void OnRequestUpdateLoadBarStatus(SplashScreenLoadEventArgs args
             RequestUpdateLoadBarStatus?.Invoke(args);
         }
 
+        /// <summary>
+        /// Event to throw for Splash Screen to display the content in the proper language
+        /// </summary>
+        internal static event SplashScreenLanguageDetected LanguageDetected;
```

Comment:
Can you remind me where is this event thrown?

Reply:
> Can you remind me where is this event thrown?

after detect the language based on the preference so the UI can display the loading messages properly and before to build the Dynamo Model

https://github.com/jesusalvino/Dynamo/blob/ff3862ccd9fd2f82f133497a7935154c851485ab/src/DynamoApplications/StartupUtils.cs#L290